src/constants/interfaces.ts

```diff
@@ -38,3 +38,25 @@ export type TooltipPlacement =
     | "leftBottom"
     | "rightTop"
     | "rightBottom";
+
+export enum ButtonClass {
+    Primary = "primary-button",
+    Secondary = "secondary-button",
+    Action = "action-button",
+}
+
```

Comment:
```suggestion
create iconGlyphClass = (name) => return `icon-moon anticon ${name}`
```

Reply:
I made that function but moved it to utils and typed the argument as `IconGlyphs` so we could keep the type safety and not use strings directly, does that capture what you were thinking?